package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "4.3.7",
+  "version": "4.3.8",
```

Comment:
This is a significant update... go for 4.4.0!

Reply:
sounds good